R/tspa.R

```diff
@@ -12,27 +12,78 @@
 #'           factor for single-group 2S-PA. A list or data frame storing
 #'           the standard errors of each group in each latent factor for
 #'           multigroup 2S-PA.
+#' @param vc A variance-covariance matrix of the latent variables, which can
+#'           be obtained from the output of \code{get_fs()} using \code{attr()}
+#'           with the argument \code{which = "av_efs"}.
+#' @param cross_loadings A matrix of loadings and cross-loadings from the
+#'                       factor scores \code{fs} to the latent variables, which
+#'                       can be obtained from the output of \code{get_fs()}
+#'                       using \code{attr()} with the argument
+#'                       \code{which = "fsA"}.
+#'                       For details see the multiple-factors vignette:
+#'                       \code{vignette("multiple-factors", package = "R2spa")}.
 #' @param ... Additional arguments passed to \code{\link[lavaan]{sem}}. See
 #'            \code{\link[lavaan]{lavOptions}} for a complete list.
 #' @return An object of class \code{lavaan}, with an attribute \code{tspaModel}
 #'         that contains the model syntax.
 #' @export
 #' @examples
 #' library(lavaan)
-#' ### single-group example
 #'
-#' # get factor scores
-#' fs_dat_ind60 <- get_fs(data = PoliticalDemocracy,
-#'                        model = "ind60 =~ x1 + x2 + x3")
-#' fs_dat_dem60 <- get_fs(data = PoliticalDemocracy,
-#'                        model = "dem60 =~ y1 + y2 + y3 + y4")
-#' fs_dat <- cbind(fs_dat_ind60, fs_dat_dem60)
+#' # single-group, two-factor example
+#' mod1 <- "
+#'    # latent variables
```

Comment:
I would revert this example back as it is still possible to estimate factor scores separately from multiple single-factor models.